deps/chakrashim/src/inspector/v8-inspector-session-impl.cc

```diff
@@ -28,7 +31,9 @@ bool V8InspectorSession::canDispatchMethod(const StringView& method) {
          stringViewStartsWith(method,
                               protocol::Console::Metainfo::commandPrefix) ||
          stringViewStartsWith(method,
-                              protocol::Schema::Metainfo::commandPrefix);
+                              protocol::Schema::Metainfo::commandPrefix) ||
+         stringViewStartsWith(method,
```

Comment:
I thought about it and decided not to mess with the dispatcher dynamically.  The TTD dispatcher will be wired up, but any attempts to use it will return an error.